app/_tutorials/add-rate-limiting-tiers-with-kong-gateway.md

```diff
@@ -38,57 +38,110 @@ tags:
+
+1. Synchronize your configuration
 
-You can define consumer groups as tiers, for example:
+   Check the differences in your files:
+   ```sh
+   deck gateway diff deck_files
+   ```
 
-* A **gold tier** consumer group with 1000 requests per minute
-* A **silver tier** consumer group with 10 requests per second
-* A **bronze tier** consumer group with 6 requests per second
+   If everything looks right, synchronize them to update your Kong Gateway configuration:
+   ```sh
+   deck gateway sync deck_files
+   ```
+
+1. Create four consumers, one for each tier and one that won't be assigned to a consumer group tier:
   
-Consumers that are not in a consumer group default to the Rate Limiting advanced plugin’s configuration, so you can define tier groups for some users and have a default behavior for consumers without groups.
+  ```yaml
+  consumers:
+  - username: Amal
+    groups:
+    - name: Free
+  - username: Dana
+    groups:
+    - name: Basic
+  - username: Mahan
+    groups:
+    - name: Premium
+  - username: Rosario
+  ```
+
+  Apphend this to your `kong.yaml` file. 
+
+  In this tutorial, one consumer won't be assigned to a tier, which is a consumer group. Consumers that are not in a consumer group default to the Rate Limiting advanced plugin’s configuration, so you can define tier groups for some users and have a default behavior for consumers without groups.
 
-## Steps
+1. Synchronize your configuration
+
+   Check the differences in your files:
+   ```sh
+   deck gateway diff deck_files
+   ```
 
-1. Enable the Rate Limiting Advanced plugin on the consumer group:
-{% capture step %}
-  {% entity_example %}
-    type: plugin
-    data:
-      name: rate-limiting-advanced
-      config:
-        limit: 5
-        window_size: 30
-        window_type: sliding
-        retry_after_jitter_max: 0
- 
-    targets:
-        - consumer_group
-
-    variables:
-      'consumerGroupName|Id': Gold
-  {% endentity_example %}
-{% endcapture %}
-{{ step | indent: 3 }}
-
-    This configuration sets the rate limit to five requests (`config.limit`) for every 30 seconds (`config.window_size`).
+   If everything looks right, synchronize them to update your Kong Gateway configuration:
+   ```sh
+   deck gateway sync deck_files
+   ```
+
+1. Enable the Rate Limiting and Rate Limiting Advanced plugins for each tier:
+  ```yaml
+  plugins:
+  - consumer_group: Free
+    config:
+      limit: 3
+      window_size: 30
+      window_type: sliding
+      retry_after_jitter_max: 0
+    name: rate-limiting-advanced
+  - consumer_group: Basic
+    config:
+      limit: 5
+      window_size: 30
+      window_type: sliding
+      retry_after_jitter_max: 0
+    name: rate-limiting-advanced
+  - consumer_group: Premium
+    config:
+      limit: 500
+      window_size: 30
+      window_type: sliding
+      retry_after_jitter_max: 0
+    name: rate-limiting-advanced
+  - name: rate-limiting
+    config:
+      second: 5
+      hour: 10000
+      policy: local
```

Comment:
Is this configured correctly? I feel like the rate limits for this one probably need to be adjusted to less than free, or do we even need this? Maybe global policy vs. policy for consumer groups is a different tutorial.

Reply:
That depends on the purpose of the tutorial, if it's just about consumer groups, you don't need the rate-limiting plugin.